static/js/components/ErrorMessage_test.js

```diff
@@ -59,10 +59,10 @@ describe("ErrorInfo", () => {
             }
```

Comment:
(as discussed in the PR description) this is just a refactor to some existing assertions. not directly related to this PR, but it was an easy enough fix and i did the same thing in a few different files